src/app/shared/filestack/filestack.component.ts

```diff
@@ -4,12 +4,14 @@ import { FilestackService } from './filestack.service';
 
 @Component({
   selector: 'file-stack',
-  templateUrl: 'filestack.component.html'
+  templateUrl: 'filestack.component.html',
+  styleUrls: ['filestack.component.scss']
 })
 export class FilestackComponent implements OnInit {
   @Input() accept: any;
   @Input() fileType: string;
   @Output() complete: EventEmitter<any> = new EventEmitter();
+  @Input() profileImage?: boolean;
```

Comment:
Instead of using `profileImage` as the parameter, it'll be better to make it more generic to other cases in the future.
Call it `type` instead, and use `'profileImage'` as the value, and use switch/case in html for different cases. "Upload" button by default